control-plane-operator/controllers/hostedcontrolplane/storage/assets/10_deployment-hypershift.yaml

```diff
@@ -0,0 +1,137 @@
+# *** AUTOMATICALLY GENERATED FILE - DO NOT EDIT ***
```

Comment:
can you ref the source? automatically generated by what/how?

Reply:
Added a README.md